### ISSUE TITLE
FeatureRequest: microphone selection

### ISSUE THREAD

**Issue by gchilds** [lihaoyun6/QuickRecorder#53]

**Comment by thekamilpro** [comment]
Yes that's a great one - which microphone is being used? Trying to use my external one, set it as default and QuickRecorder doesn't record from it

**Comment by lihaoyun6** [comment]
> Yes that's a great one - which microphone is being used? Trying to use my external one, set it as default and QuickRecorder doesn't record from it

QuickRecorder will record audio from the macOS default microphone (the one you selected in the system audio settings)

**Comment by thekamilpro** [comment]
Thanks! I got it now - it picks the default Microphone when the app lunches; hence why in my initial effort it didn't record sound, as I changed input after lunching the app :).

**Comment by SevDaisy** [comment]
The same problem bothers me. 

> Thanks! I got it now - it picks the default Microphone when the app lunches; hence why in my initial effort it didn't record sound, as I changed input after lunching the app :).

after lunching the app --- what app do you mean, the quickRecorder? 
oh yeah... seem to be a stupid question🥺

> QuickRecorder will record audio from the macOS default microphone (the one you selected in the system audio settings)
how can I make sure which one is the default microphone? is the highlight means to be default ?

**Comment by SevDaisy** [comment]
Ok, I know what happened..

**Comment by akash-joshi** [comment]
+1 love this tool and would love to see this feature incorporated into the app. Is this something I can pick up? @lihaoyun6

**Comment by lihaoyun6** [comment]
Sorry, I've been a bit busy lately.
But I saw your request and I'm already working on it.
The "Select Microphone" and "Record System Sound + Microphone Only" features are coming soon, you will get them in the next version!

**Comment by lihaoyun6** [comment]
<img width="278" alt="image" src="https://github.com/lihaoyun6/QuickRecorder/assets/16348097/8bbf9ec8-51b7-46e5-888e-81e4d5e87cb8">
<img width="400" alt="image" src="https://github.com/lihaoyun6/QuickRecorder/assets/16348097/1e128ba7-a876-4331-bf81-8f1ae9518bcb">  

With the new `.qma` multi-track audio format, the upcoming version of QuickRecorder will allow users to record system sound and microphone at the same time.  
After recording, you can also use the built-in qma player to modify the volume of any track independently at any time; or export it to a common audio format with a single track.  

在即将发布的新版本中, 利用新的 `.qma` 多轨音频格式, QuickRecorder允许用户同时录制系统声音和麦克风.  
并且录制结束后还可以随时用内置的qma播放器独立修改任何一轨的音量; 或者导出成单音轨的普通音频格式.

**Comment by lihaoyun6** [comment]
This feature has been implemented in v1.4.1. Please update to get all new features.  
此特性已在v1.4.1版本中正式实装. 请尝试更新以获取全部新功能.
